libraries/AP_RPM/AP_RPM.cpp

```diff
@@ -25,7 +26,7 @@ const AP_Param::GroupInfo AP_RPM::var_info[] = {
     // @Param: _TYPE
     // @DisplayName: RPM type
     // @Description: What type of RPM sensor is connected
-    // @Values: 0:None,1:PWM,2:AUXPIN,3:EFI
+    // @Values: 0:None,1:PWM,2:AUXPIN,3:EFI,4:Harmonic Notch
```

Comment:
it isn't really "harmonic notch" providing the RPM, it is a FFT

Reply:
Actually not quite - that's the point - if the throttle estimate or ESC telemetry are driving the harmonic notch then you see those. That's what's so cool - you almost always get an understandable result regardless of the underlying driver.